daprdocs/content/en/operations/hosting/kubernetes/kubernetes-dapr-shared.md

```diff
@@ -0,0 +1,86 @@
+
+
+<img src="/images/dapr-shared/sidecar.png" width=800 style="padding-bottom:15px;">
+
+While sidecars are Dapr's default deployment strategy, some use cases require other approaches. Let's say you want to decouple the lifecycle of your workloads from the Dapr APIs. A typical example of this is functions, or function-as-a-service runtimes, which might automatically downscale your idle workloads to free up resources. For such cases, keeping the Dapr APIs and all the Dapr async functionalities (such as subscriptions) separate might be required. 
+
+Dapr Shared was created for these scenarios, extending the Dapr sidecar model with two new deployment approaches: `DaemonSet` and `Deployment`.
+
+{{% alert title="Important" color="primary" %}}
+No matter which strategy you choose, it is important to understand that in most use cases, you have one instance of Dapr Shared (Helm release) per service (app-id). This means that if you have an application composed of three microservices, each service is recommended to have its own Dapr Shared instance. You can see this in action by trying the [Hello Kubernetes with Dapr Shared tutorial](https://github.com/dapr/dapr-shared/blob/main/docs/tutorial/README.md). 
+{{% /alert %}}
+
+
+### `DeamonSet` strategy
+
+With Kubernetes `DaemonSet`, you can define workloads that need to be deployed once per node in the cluster. This enables workloads that are running on the same node to communicate with local Dapr APIs, no matter where the Kubernetes `Scheduler` schedules your workload.
+
+<img src="/images/dapr-shared/daemonset.png" width=800 style="padding-bottom:15px;">
+
+{{% alert title="Note" color="primary" %}}
+Since `DaemonSet` installs one instance per node, it consumes more overall resources in your cluster, compared to `Deployment` for a per cluster deployment.
+{{% /alert %}}
+
+
+### `Deployment` strategy
+
+Kubernetes `Deployments` are installed once per cluster. Based on available resources, the Kubernetes `Scheduler` decides on which node the workload is scheduled. For Dapr Shared, this means that your workload and the Dapr instance might be located on separate nodes, which can introduce considerable network latency with the trade-off of reduce resource usage.
+
+<img src="/images/dapr-shared/deployment.png" width=800 style="padding-bottom:15px;">
+
+## Getting Started with Dapr Shared
+
+{{% alert title="Prerequisites" color="primary" %}}
+Before installing Dapr Shared, make ensure you have [Dapr installed in your cluster]({{< ref "kubernetes-deploy.md" >}}).
+{{% /alert %}}
+
+If you want to get started with Dapr Shared, you can create a new Dapr Shared instance by installing the official Helm Chart:
+
+```
+helm install my-shared-instance oci://registry-1.docker.io/daprio/dapr-shared-chart --set shared.appId=<DAPR_APP_ID> --set shared.remoteURL=<REMOTE_URL> --set shared.remotePort=<REMOTE_PORT> --set shared.strategy=deployment
```

Comment:
I still do not follow this. Where do you specify per-node verses per-cluster in the helm chart?
Also what should you put for the Remote URL and Remote port typically? What is the guidance here?
And do those ports relate to what you have to put in the port values for the SDK Env Variables?

Reply:
for this: `Where do you specify per-node versus per-cluster in the helm chart?`  you set this variable 
`--set shared.strategy=deployment` or `--set shared.strategy=daemonset`

for this: `Also what should you put for the Remote URL and Remote port typically?`

This are your application details, where is the application running and which port can be used by Dapr to callback the app. This will be usually a Kubernetes Service URL .. like myservice.default.svc.cluster.local and the port that the application is using, most of the time services expose the port 80. 

for this: `And do those ports relate to what you have to put in the port values for the SDK Env Variables?`

Not at all, the ports that you put in the SDK are always the same, in the SDK you only reference to the services that is created by the Dapr Shared instance.